deployments/salt/README.md

```diff
@@ -140,20 +140,35 @@ splunk-otel-collector:
 
 ### Auto Instrumentation (Linux Only)
 
-**Note:** The Java application(s) on the node need to be started/restarted
-separately after installation/configuration in order for any changes to take
-effect.
+**Note:** The application(s) on the node need to be started/restarted separately
+after installation/configuration in order for any changes to take effect.
 
 - `install_auto_instrumentation`: Whether to install/manage [Splunk
-  OpenTelemetry Auto Instrumentation for Java](
+  OpenTelemetry Auto Instrumentation](
   https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation).
   When set to `True`, the `splunk-otel-auto-instrumentation` deb/rpm package
   will be downloaded and installed from the Collector repository. (**default:**
   `False`)
 
 - `auto_instrumentation_version`: Version of the
   `splunk-otel-auto-instrumentation` package to install, e.g. `0.50.0`. The
-  minimum supported version is `0.48.0`. (**default:** `latest`)
+  minimum supported version is `0.48.0`. The minimum supported version for
+  Node.js auto instrumentation is `0.87.0`. The minimum supported version for
+  .NET auto instrumentation is `0.99.0`. (**default:** `latest`)
```

Comment:
> Does this need to be updated since collector `v0.100.0` was released? Or is this not the version of the collector?

This is the version for the `splunk-otel-auto-instrumentation` package.  `0.99.0` is the first version of this package that includes `splunk-otel-dotnet`.